pkg/utils/release/pipelines.go

```diff
@@ -0,0 +1,67 @@
+package release
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Contains all methods related with pipeline objects CRUD operations.
+type PipelinesInterface interface {
```

Comment:
this methods should be in release interface

Reply:
+1, we don't need another pipeline file

Reply:
Moved.